### PR TITLE
Fix incorrect units in dtend metadata

### PR DESCRIPTION
**Pull request by SamuelTrahanNOAA**
Several units in dtend output variables had the wrong units (kg/(kg*s)). This is now corrected.

Corresponding fv3atm PR:

* https://github.com/NOAA-GSL/fv3atm/pull/82

This error was discovered during review of the documentation PR:

* https://github.com/NCAR/ccpp-doc/pull/38

Which was reviewed by @ligiabernardet

----------

**Note:** This PR must not be merged until the submodule for FV3 is pointed back to NOAA-GSL. It points to SamuelTrahanNOAA so the branch can be recursively cloned and tested.

### CODE REVIEW

**Comment by SamuelTrahanNOAA** [comment]
This PR needs a new baseline due to the different metadata in NetCDF output files. The baseline is here:

* Hera: /scratch2/BMC/wrfruc/Samuel.Trahan/hydro-tend/FV3_RT/develop-20210330

**Comment by SamuelTrahanNOAA** [comment]
I need someone's approval of this PR to update the FV3 submodule to the top of gsl/develop. Until then, gsl/develop will be newer than its submodule hash. Pinging @DomHeinzeller and @ligiabernardet.

I'll be gone most of today, so if it is approved, then it would be great if someone else merges in my absence.

**Review by DomHeinzeller — APPROVED** [review]

**Pull request merged**
